src/components/clusterConfiguration/NetworkConfiguration.tsx

```diff
@@ -42,13 +41,11 @@ const NetworkConfiguration: React.FC<NetworkConfigurationProps> = ({
     setAdvanced(checked);
 
     if (!checked) {
-      setFieldValue('clusterNetworkCidr', CLUSTER_DEFAULT_NETWORK_SETTINGS.clusterNetworkCidr);
-      setFieldValue(
-        'clusterNetworkHostPrefix',
-        CLUSTER_DEFAULT_NETWORK_SETTINGS.clusterNetworkHostPrefix,
-      );
-      setFieldValue('serviceNetworkCidr', CLUSTER_DEFAULT_NETWORK_SETTINGS.serviceNetworkCidr);
-      setFieldValue('additionalNtpSource', CLUSTER_DEFAULT_NETWORK_SETTINGS.additionalNtpSource);
```

Comment:
additionalNtpSource is now part of advanced network fields, so it needs to get reset as well